### PR TITLE
Powersystem

### LINKED ISSUE #96: Make the battery percentage more intuitive

### PR DESCRIPTION
**Pull request by wang-henry**
Closes #96 #80 #99

### CODE REVIEW

File: Assets/Scripts/Tutorial-Power/TutorialGuide.cs [hunk 1]
```diff
@@ -25,16 +25,14 @@ public class TutorialGuide : MonoBehaviour
...
-    public GameEvent disableDrain;
-
     // Enable and disable switch function
     public GameEvent enableSwitch;
     public GameEvent disableSwitch;
 
     [Header("Listening To")]
     public GameEvent switchOn;
     public GameEvent powerOn;
+    public GameEvent powerOff;
```
Comment (ectodrop) [inline Assets/Scripts/Tutorial-Power/TutorialGuide.cs:35]: This isn't set in the inspector and gives a null reference exception

**Review by ectodrop — COMMENTED** [review]
Looks great!
Getting this error on the percentbar and ParcentBarBackground, I think all you have to do is delete the mesh collider from these objects.
image

**Comment by ectodrop** [comment]
Let's also change the color of the battery to be more battery like, look at a duracell battery for reference

File: Assets/Scripts/Tutorial-Power/TutorialGuide.cs (hunk 1 above)
Comment (wang-henry) [reply Assets/Scripts/Tutorial-Power/TutorialGuide.cs:35]: Sorry I realized I forgot to save the scene.

**Review by ectodrop — APPROVED** [review]
Thanks!

**Comment by wang-henry** [comment]
Closes [#88](https://github.com/ectodrop/loop-game/issues/88)

**Pull request merged**